isx/__init__.py

```diff
@@ -192,11 +192,12 @@ def get_frame_data(self, index: int):
                 "Unknown number of bytes per pixel. Cannot decode this frame."
             )
 
+        header_offset = 0
+        footer_offset = 0
         if self.footer["hasFrameHeaderFooter"]:
-            raise NotImplementedError(
-                """[UNIMPLEMENTED] Cannot extract frame from this
-        movie because frames have footers and headers."""
-            )
+            header_offset = (index + 1) * bytes_per_pixel * 1280 * 2
```

Comment:
please make `1280 * 2` a constant called something like `header_footer_num_pixels` wth a comment clarifying what it is for future devs.

also `bytes_per_pixel` is not need here because the header/footer is independent of the image data type. it will always be the same number of bytes regardless of image data type

I suggest the following
```suggestion
            header_footer_size_in_bytes = 1280 * 2 * 2 # The size of the frame header in bytes. The same for the footer.
            header_offset = (index + 1) * header_footer_size_in_bytes
            footer_offset = (index + 1) * header_footer_size_in_bytes            
```